src/Discord.Net.Commands/Info/CommandInfo.cs

```diff
@@ -63,6 +63,9 @@ internal CommandInfo(CommandBuilder builder, ModuleInfo module, CommandService s
             _action = builder.Callback;
         }
 
+        public TPrecondition GetPrecondition<TPrecondition>() where TPrecondition : PreconditionAttribute =>
```

Comment:
You can if you set `AllowMultiple = true`, but whether that's actually useful is a completely different thing.

Also, for the query itself, wouldn't it be better to use `x => x is TPrecondition`?